irc.html

```diff
@@ -10,24 +10,26 @@ <h1>Channels</h1>
 <div>
   <p>
     Join one of our <abbr title="Internet Relay Chat">IRC</abbr> channels on Freenode:<br/>
+    To chat on #minetest-hub you need to ask an operator for voice. You can't use webchat on #minetest-dev.<br>
   </p>
   <ul>
     <li>
       <strong><a href="https://webchat.freenode.net/?channels=#minetest">#minetest</a></strong>
-      (<a href="http://irc.minetest.net/minetest">logs</a>): main channel
+        (<a href="http://irc.minetest.net/minetest">Logs</a>):
+        Main channel for general Minetest engine, modding and games discussion or support.
     </li>
+    
     <li>
-      <strong>
-        <a href="https://webchat.freenode.net/?channels=#minetest-dev">#minetest-dev</a>
-      </strong>
-      (<a href="http://irc.minetest.net/minetest-dev">logs</a>):
-      development channel, only serious stuff; note that webchat is muted due to too much spam/flood.
+      <strong><a href="https://webchat.freenode.net/?channels=#minetest-dev">#minetest-dev</a></strong>
+        (<a href="http://irc.minetest.net/minetest-dev">Logs</a>):
+        Development channel, strictly only for discussions of <a href="https://github.com/minetest">official projects'</a> development.
```

Comment:
Hmm actually, linking to the homepage and calling that 'official projects' isn't descriptive, the home page doesn't have an 'official projects' section and what is official isn't obvious.
How about no link and describing what can be discussed (MTEngine, MTGame), as before?

Reply:
What was the old description again? Will see if I can find it easily

Reply:
`..., strictly only for development discussion of the Minetest engine and Minetest Game'
'discussion' doesn't need to be plural.

Reply:
So, what about one of the description that we came with before.
```
Development channel, for topics related to bug reporting and general Minetest development.
```